docs/remote/containers.md

```diff
@@ -19,13 +19,25 @@ This lets VS Code provide a **local-quality development experience** — includi
 
 ## Getting started
 
+### System Requirements
+
+**Local:** Docker Desktop 2.0+ for macOS/Windows or Docker CE/EE 18.06+ and Docker Compose 1.21+ for Linux. Docker Toolbox and Ubuntu snap packages are not supported. For Windows, Windows 10 Professional or Enterprise is required due to the requirements of Docker Desktop or Windows. See the minimum requirements for [VS Code](/docs/supporting/requirements.md) for additional details.
```

Comment:
Compose 1.21 is only needed when the project folder name contains hyphens or underscores.

Reply:
Yeah I thought it would be easier just to recommend 1.21 -- and all of our "try" and dev contaienr defintion test projects end up having a dash in them because of the folder name.  We're also all on latest, so there's potentially other issues we just don't know about yet.